packages/override-tools/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Tooling to manage Javascript file overrides in React Native Windows",
   "private": true,
   "license": "MIT",
+  "main": "lib-commonjs/Cli.js",
```

Comment:
This isn't how CLI's are done in node.

You should have a bin entry in your package.json, which points to this file.

Then ensure that the top of that file has a comment:
`#!/usr/bin/env node`